pubtools/_quay/operator_pusher.py

```diff
@@ -677,7 +751,7 @@ def push_index_images(self, iib_results, tag_suffix=None):
             )
 
             ContainerImagePusher.run_tag_images(
-                build_details.index_image, [dest_image], True, index_image_ts
+                build_details.index_image, dest_images, True, index_image_ts
             )
             if tag_suffix:
                 dest_image = "{0}:{1}-{2}".format(index_image_repo, tag, tag_suffix)
```

Comment:
Could you update the tag here as well? It's got from build_details.index_image right now.